src/components/Comments/ReplyCommentItem.js

```diff
@@ -190,7 +206,7 @@ const ReplyCommentItem = ({
   const onPressDMAnon = async () => {
     try {
       setLoading({...loading, loadingDmAnon: true});
-      await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS');
+      await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS', feedId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendMessageDM` function call has been updated to include the `feedId`. This change is necessary for the new feature being implemented.
```diff
+ await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS', feedId);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->